pkg/netpol/eval/internal/k8s/adminnetpol.go

```diff
@@ -185,9 +412,15 @@ func ruleConnections(ports *[]apisv1a.AdminNetworkPolicyPort, dst Peer) (*common
 			}
 			portSet.AddPort(intstr.FromInt32(anpPort.PortNumber.Port))
 		case anpPort.NamedPort != nil:
+			if dst.PeerType() == IPBlockType {
+				// IPblock does not have named-ports defined
+				// @tbd should return error? (a rule that combines networks and pods may have such port?)
```

Comment:
as for "tbd" comment: can add an example test for this case as well?
if a consistent connectivity output is possible, a warning is preferred.

Reply:
done; added test `anp_test_named_ports_multiple_peers` with a warning that the named-port is ignored for an IP-Block